samples/artifacts/DataSummarization/NamedAggregation.siddhi

```diff
@@ -0,0 +1,67 @@
+@App:name("NamedAggregation")
+@App:description("Demonstrates how to create a named aggregation for better query performance.")
+
+/*
+Purpose:
+    This application demonstrates how to create the named aggregation for better query performance.In this example the named aggregation keeps the aggregation of events such as average of price and sum of price.For more information on windows please refer to "https://siddhi.io/siddhi/documentation/siddhi-5.0/#window" The 'group by' clause helps to perform aggregation on events grouped by a certain attributes.In this sample, the symbol information per items is aggregated.
```

Comment:
Not the real use case of aggregation, as comments for the window cannot be reused. Review

Reply:
0c298022e90d12e96fa96a6435c301871fc28edb This commit fixed the above correction